spec/index.html

```diff
@@ -273,17 +273,19 @@ <h3>Serializing the Results Table</h3>
       <h3>Serializing RDF Terms</h3>
       <p>The SPARQL Results TSV Results Format serializes RDF terms in the results table by using the syntax that SPARQL
        [<a href="#SPARQL11-QUERY">SPARQL11-QUERY</a>] and Turtle [<cite><a class="bibref" rel="biblioentry" href="#bib-TURTLE">TURTLE</a></cite>] use.</p>
-      <p>IRIs enclosed in <code>/&lt;...&gt;</code>, literals are enclosed with double quotes <code>/"</code>...<code>/"</code> or single quotes <code>/'</code> ...<code>/'</code> with optional <code>/@lang</code> or <code>/^^</code>
-      for datatype. The quotes around the lexical form is required. Tab, newline and carriage return characters (Unicode codepoints 0x09, 0x0A (line feed) and 0x0D (Carriage Return)) are encoded in
-      strings as <code>/\t</code>, <code>/\n</code> and <code>/\r</code> respectively. The long string forms using triple quotes <code>/"""</code> and <code>/'''</code> <em class="rfc2119" title="must not">must not</em> be
+      <p>IRIs are enclosed in <code>&lt;...&gt;</code>, literals are enclosed with double quotes <code>"</code>...<code>"</code> or single quotes <code>'</code> ...<code>'</code> with optional <code>@lang</code> or <code>^^</code>
+      for datatype.
+      IRIs <em class="rfc2119" title="must">must</em> be absolute, and datatypes <em class="rfc2119" title="must">must</em> be written without prefixes in their absolute form enclosed in <code>&lt;...&gt;</code>.
+      The quotes around the lexical form are required. Tab, newline and carriage return characters (Unicode codepoints 0x09, 0x0A (line feed) and 0x0D (Carriage Return)) are encoded in
+      strings as <code>\t</code>, <code>\n</code> and <code>\r</code> respectively. The long string forms using triple quotes <code>"""</code> and <code>'''</code> <em class="rfc2119" title="must not">must not</em> be
```

Comment:
An idea while reading this MR (please ignore): it might be nice to also specify what to do with other "special" characters (`\0`, `"`, `\`...), with maybe an harmonized approach with canonical N-Quads.

Reply:
@tpt Please could you raise that as an issue on this document? Thanks.

Reply:
Done #10 10